### DISCUSSION TITLE
Hide source code from react app

### DISCUSSION THREAD

**Discussion by eneadriancatalin** [vercel/community#2019] [Help] [answered]
### Summary

Hello guys, I'm new in this web dev world and I'm still learning so be kind :)
I did a website for a good friend and now I deployed it on vercel, all good by now. But when I go into dev tools -> sources I can see a static folder with all my components and app.js files, how can I hide them? I'm not sure if I deployed it correctly (meaning if I deployed the build folder)

https://photography-website-eta.vercel.app/

**Comment by swarnava** [comment, marked as answer]
You can find the instruction here: https://www.dhiwise.com/post/how-to-hide-javascript-source-code
